infinity-app/app.R

```diff
@@ -118,7 +161,7 @@ ui <-
         inputId = "taxa_genus",
         label = "Choose a genus: (you can also select All, but it's slow so be patient)",
         choices = "Eucalyptus",
-        selected = "Eucalyptus",
+        selected = "All",
```

Comment:
I'm also confused by this when I load the app, the default is Sclerolaena and not all...

Reply:
all defaults changed to "All"